lib/Email/MIME.pm

```diff
@@ -119,7 +119,15 @@ my $NO_ENCODE_RE = qr/
 /ix;
 
 sub new {
-  my $self = shift->SUPER::new(@_);
+  my ($class, $text, $arg, @rest) = @_;
+  $arg ||= {};
+
+  my $encode_check = exists $arg->{encode_check}
+                   ? delete $arg->{encode_check}
+                   : 1; # Encode::FB_CROAK
```

Comment:
Rather use that `Encode::FB_CROAK` named constant. `1` is magic number, `FB_CROAK` makes code more readable. Same for documentation. People doing copy+paste of examples, so `FB_CROAK` resp. `FB_DEFAULT` would make their code more readable too...